seatunnel-examples/seatunnel-flink-examples/src/main/resources/examples/fake_to_console.conf

```diff
@@ -23,6 +23,11 @@ env {
   execution.parallelism = 1
   #execution.checkpoint.interval = 10000
   #execution.checkpoint.data-uri = "hdfs://localhost:9000/checkpoint"
+  execution.metrics.class = org.apache.seatunnel.metrics.core.reporter.PrometheusPushGatewayReporter
```

Comment:
Not add this in default config, because other developer environment don't have Prometheus.